build.wake

```diff
@@ -161,10 +161,20 @@ export def static _: Result Path Error =
     | getJobOutput
 
 from test_wake import topic wakeTestBinary
+from test_wake import topic wakeUnitTestBinary
 publish wakeTestBinary = defaultWake, Nil
+publish wakeUnitTestBinary = defaultWakeUnit, Nil
```

Comment:
Why do we need this sort of pub-sub indirection here?
Couldn't `test_wake` just import `buildWakeUnit` from `build_wake`?

Reply:
I was actually just cargo culting the pattern you used for wakeTestBinary until I got to a place where I understood what was happening honestly. Much happier to do it that way.